### PR TITLE
Update katello-host-tools to 3.4.0

### PR DESCRIPTION
**Pull request by johnpmitsch**
Update katello-host-tools to 3.4.0 and specify
subscription-manager version

For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -30,7 +30,7 @@ ExclusiveArch: x86_64
 BuildArch: noarch
 %endif
 
-Requires: subscription-manager
+Requires: subscription-manager >= 1.24
```
Comment (parthaa) [inline packages/client/katello-host-tools/katello-host-tools.spec]: Safer to do >= 1.23 for rhel

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 2]
```diff
@@ -145,8 +145,6 @@ rm -rf %{buildroot}
 
 %if %{dnf_install}
 %global katello_libdir %{python3_sitelib}/katello
```
Comment (johnpmitsch) [inline packages/client/katello-host-tools/katello-host-tools.spec]: @jturel Is this directory still valid?

**Comment by jturel** [comment]
I wanted to test this before merge :(

**Comment by johnpmitsch** [comment]
@ehelms was there a reason for merging this? People were reviewing it and we were waiting for confirmation on the subscription-manager version from candlepin

**Comment by jturel** [comment]
Not the end of the world. It occurred to me that we should probably preserve the DNF tracer plugin for fedora clients which is removed in this version. I've opened https://github.com/Katello/katello-host-tools/pull/86 to bring the code back. I'll open another packaging PR when it's time and do the testing that I wanted to for this one

**Comment by ehelms** [comment]
@jturel you still can! but please try to mention that before hand or have submitter put it into WIP state :) We can always rebuild or update.

**Comment by ehelms** [comment]
I have not tagged this into 1.20 so its just in nightly.

**Comment by jturel** [comment]
@ehelms totally fair. I think it's also fair to mention there were no ACKs on the PR!

**Comment by johnpmitsch** [comment]
>  have submitter put it into WIP state

@ehelms Does this repository not require ACKs/Approvals to merge? It doesn't seem like our typical process to merge a PR that no one has approved. I also have merge rights here and was planning to merge once approved by those involved in this change. I'm not sure what I am missing.

**Comment by ehelms** [comment]
@johnpmitsch @jturel Fair, sometimes I forget to ACK and just merge (like an implied ACK!) :)

I'll do better.

**Pull request merged**